Tools/clinic/clinic.py

```diff
@@ -5033,26 +5033,27 @@ def bad_node(self, node):
         key = f"{parameter_name}_as_{c_name}" if c_name else parameter_name
         self.function.parameters[key] = p
 
-    KwargDict = dict[str | None, Any]
+    KwargDict = dict[str, Any]
 
     @staticmethod
     def parse_converter(annotation: ast.expr | None) -> tuple[str, bool, KwargDict]:
+        msg = "Annotations must be either a name, a function call, or a string."
         match annotation:
             case ast.Constant(value=str() as value):
                 return value, True, {}
             case ast.Name(name):
                 return name, False, {}
             case ast.Call(func=ast.Name(name)):
                 symbols = globals()
-                kwargs = {
-                    node.arg: eval_ast_expr(node.value, symbols)
-                    for node in annotation.keywords
-                }
+                kwargs: dict[str, Any] = {}
```

Comment:
Why can't I use `KwargDict` here?

Reply:
It's not in scope, right? Code blocks in methods can't see variables in the enclosing class, and `KwargDict` is defined in the scope of the enclosing class currently. If you defined the type alias in the module scope, you could use it here

Reply:
Probably. Let's move it.

Reply:
See bb879ccebd9fe5ba5fb2bbb8168c550c00d6a5dd